app/services/invoices/preview_context_service.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+module Invoices
+  class PreviewContextService < BaseService
+    def initialize(organization, params:)
+      @organization = organization
+      @params = params.presence || {}
+      super(nil)
```

Comment:
This should be enough:
```suggestion
      super
```

Reply:
That was needed since `.initialize` accepted not only keyword parameters. I've made those consistent and removed `(nil)`.